index.html

```diff
@@ -388,14 +388,16 @@ <h2>
                 <div class="btn_container">
                 <button role="button" aria-label="Get in touch" type="submit" class="btngreen">Get in touch</button>
                 </div>    
+
```

Comment:
- [ Optional] Well done implementing the form as required, as you can see in the caption , the font is used for the text is different from the one used in Figma design. I recommend using the same font so that your project will look perfect.
Your design
<img width="401" alt="Screenshot 2023-05-06 at 11 50 44 PM" src="https://user-images.githubusercontent.com/53347966/236649713-7318f0e4-641f-42c2-9c73-44acdbf9e465.png">